pterodactyl.php

```diff
@@ -239,6 +238,9 @@ public function addService(
             return;
         }
 
+        // Set configurable options
+        $package = $this->getConfigurableOptions($vars, $package);
```

Comment:
What about edit? Or will that be included in CORE-4123?

Reply:
Yes, to be able to modify the configurable options it is necessary that the package can be modified, so it seems that it is more suitable for CORE-4123.